backend/scripts/seed-local.sql

```
@@ -111,21 +112,37 @@ SELECT
+  '$2a$10$Xgxiol1/CKW0E2qp4P3JOO/fZp3dcDmXxMHk76rHrOLRec8RIaqEm',
   true,
   now() - (random() * 365)::int * interval '1 day',
   now()
 FROM generate_series(1, :n_users) AS i;
 
+-- オフラインで管理画面まで触れるよう、運営管理者を 1 人入れる（admin@example.test / password）。
+-- id 1000000 は連番（1000000 + i, i >= 1）と衝突しない。
+INSERT INTO users (id, cognito_sub, email, name, company_id, role, role_id, password_hash, is_active, created_at, updated_at)
+VALUES (
+  1000000, 'seed-sub-admin', 'admin@example.test', 'シード運営管理者', NULL,
+  'super_admin', (SELECT id FROM roles WHERE name = 'super_admin'),
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)